types/part_set.go

```diff
@@ -131,10 +131,10 @@ func NewPartSetFromData(data []byte, partSize int) *PartSet {
 // Returns an empty PartSet ready to be populated.
 func NewPartSetFromHeader(header PartSetHeader) *PartSet {
 	return &PartSet{
-		total:         header.Total,
+		total:         int(header.Total),
```

Comment:
any risk of overflow on 32 bit systems?

Reply:
i would like to discuss this issue: https://github.com/tendermint/tendermint/issues/2684 in todays call. it will bring to light how to move forward with int types

Reply:
also do claim support for 32 bit systems?

Reply:
we do, currently